Modules/CalcPerform.lua

```diff
@@ -425,6 +425,19 @@ local function doActorMisc(env, actor)
 			end
 			modDB.multipliers["BuffOnSelf"] = (modDB.multipliers["BuffOnSelf"] or 0) + 1
 		end
+		if modDB:Sum("BASE", nil, "DamageFromHitsTakenFromSpectres") > 0 then
+			local isActiveSpectre = false
+			for _, activeSkill in ipairs(env.player.activeSkillList) do
+				if activeSkill.skillFlags.spectre then
+					isActiveSpectre = true
+				end
+			end
```

Comment:
This loop is already done on https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/3e28ad5599ebcddf8377cb3cbb433fefe4fb7ff4/Modules/CalcPerform.lua#L767-L770

~~Can you change this code to use that variable, instead?~~

Actually, on second thought, you can do this entirely in ModParser.lua, see that for a suggestion